CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # App Center SDK for Unity Editor Extensions Change Log
 
+## Development version
+
+* **[Bug fix]** Fix call to github on every time when editor enters play mode. 
```

Comment:
Can we rephrase this? Not sure to understand the wording.

Reply:
Fixed